tests/test_utils/test_nan_warnings.py

```diff
@@ -0,0 +1,77 @@
+"""
+Test the RadViz feature analysis visualizers
+"""
+
+import unittest
+import numpy as np
+
+from yellowbrick.utils.nan_warnings import warn_if_nans_exist, count_nan_rows, \
+    drop_rows_containing_nans
+
+
+class TestNanWarnings(unittest.TestCase):
+    def test_raise_warning_if_nans_exist(self):
+        """
+        Test that a warning is raised if any nans are in the data
+        """
+        data = np.array([
+            [1, 2, 3],
+            [1, 2, np.nan],
+        ])
+
+        self.assertWarns(UserWarning, warn_if_nans_exist, data)
+
+    def test_count_rows_with_nans(self):
+        """
+        Test that a warning is raised if any nans are in the data
+        """
+        data0 = np.array([
+            [1, 2, 3],
+            [1, 2, 3],
+            [1, 2, 3],
+        ])
+
+        data2 = np.array([
+            [np.nan, 2, 3],
+            [1, np.nan, 3],
+        ])
+
+        data3 = np.array([
+            [np.nan, 2, 3],
+            [1, np.nan, 3],
+            [np.nan, np.nan, np.nan],
+        ])
+
+        self.assertEqual(0, count_nan_rows(data0))
```

Comment:
Same here, since we're moving away from unittest, you can just do assert `count_nan_rows(data0) == 0`